models/git_diff.go

```diff
@@ -550,7 +550,13 @@ func ParsePatch(maxLines, maxLineCharacters, maxFiles int, reader io.Reader) (*D
 			beg := len(cmdDiffHead)
 			a := line[beg+2 : middle]
 			b := line[middle+3:]
+
 			if hasQuote {
+				// When /a and /b are surrounded by double quotes, we want to first
+				// make sure we keep everything the quotes and then strip out the leading /a /b
+				a = strings.Replace(line[beg:middle], "\"a/", "\"", -1)
```

Comment:
Why don't we just unquote then remove the first two runes. Using replace here is just complicating things.

Consider the, admittedly unusual case, of a file called: `/a/a/a/a` which is `git mv` and editted to `/b/b/b/b`. Your replace will just remove the entire names. (Yes you could change the replace to only replace one time but removing the first two characters is your aim.)

Reply:
Ah yea that makes more sense/less complicated thx. Slipped my mind to change the string after unquoting. I've changed it to just remove the first 2 characters afterwards as you've suggested thx!

I agree entire integration tests are a good goal but I don't want to complicate or hold up what is otherwise just a couple line bug fix + adding a unit test here.

 I'll look into understanding the integration testing more and perhaps help with something in a separate request if I can.